src/poggit/ci/builder/DefaultProjectBuilder.php

```diff
@@ -195,6 +199,65 @@ protected function build(Phar $phar, RepoZipball $zipball, WebhookProjectModel $
         return $result;
     }
 
+    private function runDynamicCommandList(string $pharPath, string $pluginName, int $buildId){
+        if($pluginName === null){
+            Meta::getLog()->e("Failed to start dyncmdlist, no plugin name found.");
+            return;
+        }
+
+        $id = "dyncmdlist-" . substr(Meta::getRequestId() ?? bin2hex(random_bytes(8)), 0, 4) . "-" . bin2hex(random_bytes(4));
+
+        Meta::getLog()->v("Starting dyncmdlist flow with ID '{$id}'");
+
+        Lang::myShellExec("docker run --name {$id} --cpus=1 --memory=256M -v ${pharPath}:/input/plugin.phar pmmp/dyncmdlist:0.1.1 ./wrapper.sh ${pluginName}", $stdout, $stderr, $exitCode);
```

Comment:
Is `--rm` missing here?

Reply:
Yes, removed it for debugging earlier.....

Reply:
Also just to confirm, is `${pluginName}` shell escape safe?

Reply:
Nope, straight out of plugin.yml iirc